cnxarchive/sql/search/title.part.sql

```diff
@@ -10,4 +10,4 @@ SELECT
 FROM
   latest_modules
 WHERE
-  name ~* req(%({0})s::text)
+  StripHTML(name) ~* req(%({0})s::text)
```

Comment:
I'm not so concerned with making that change on the python side.  I'm just concerned that a lot of the sql code would have to be changed.